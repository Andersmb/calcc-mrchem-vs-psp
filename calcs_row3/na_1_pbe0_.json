{
  "input": {
    "molecule": {
      "cavity_coords": [
        {
          "center": [
            0.0,
            0.0,
            -2.9091386749999995
          ],
          "radius": 2.23
        },
        {
          "center": [
            0.0,
            0.0,
            2.9091386750000003
          ],
          "radius": 2.23
        }
      ],
      "cavity_width": 0.2,
      "charge": 0,
      "coords": [
        {
          "atom": "na",
          "xyz": [
            0.0,
            0.0,
            -2.9091386749999995
          ]
        },
        {
          "atom": "na",
          "xyz": [
            0.0,
            0.0,
            2.9091386750000003
          ]
        }
      ],
      "multiplicity": 1
    },
    "mpi": {
      "bank_size": 1,
      "numerically_exact": false,
      "shared_memory_size": 10000
    },
    "mra": {
      "basis_order": 9,
      "basis_type": "interpolating",
      "boxes": [
        2,
        2,
        2
      ],
      "corner": [
        -1,
        -1,
        -1
      ],
      "max_scale": 20,
      "min_scale": -5
    },
    "printer": {
      "file_name": "na_1_pbe0_.inp",
      "print_level": 0,
      "print_mpi": false,
      "print_prec": 6,
      "print_width": 75
    },
    "rsp_calculations": {},
    "scf_calculation": {
      "fock_operator": {
        "coulomb_operator": {
          "poisson_prec": 1e-06,
          "shared_memory": true
        },
        "exchange_operator": {
          "exchange_prec": -1.0,
          "poisson_prec": 1e-06
        },
        "kinetic_operator": {
          "derivative": "abgv_55"
        },
        "nuclear_operator": {
          "proj_prec": 1e-06,
          "shared_memory": true,
          "smooth_prec": 1e-06
        },
        "xc_operator": {
          "shared_memory": false,
          "xc_functional": {
            "cutoff": 0.0,
            "functionals": [
              {
                "coef": 1.0,
                "name": "pbe0"
              }
            ],
            "spin": true
          }
        }
      },
      "initial_guess": {
        "file_CUBE_a": "cube_vectors/CUBE_a_vector.json",
        "file_CUBE_b": "cube_vectors/CUBE_b_vector.json",
        "file_CUBE_p": "cube_vectors/CUBE_p_vector.json",
        "file_basis": "initial_guess/mrchem.bas",
        "file_chk": "checkpoint/phi_scf",
        "file_gto_a": "initial_guess/mrchem.moa",
        "file_gto_b": "initial_guess/mrchem.mob",
        "file_gto_p": "initial_guess/mrchem.mop",
        "file_phi_a": "initial_guess/phi_a_scf",
        "file_phi_b": "initial_guess/phi_b_scf",
        "file_phi_p": "initial_guess/phi_p_scf",
        "localize": true,
        "method": "DFT (PBE0)",
        "prec": 0.0001,
        "relativity": "None",
        "restricted": false,
        "screen": 12.0,
        "type": "sad_gto",
        "zeta": 0
      },
      "properties": {
        "dipole_moment": {
          "dip-1": {
            "operator": "h_e_dip",
            "precision": 1e-06,
            "r_O": [
              0.0,
              0.0,
              0.0
            ]
          }
        }
      },
      "scf_solver": {
        "checkpoint": false,
        "derivative": "abgv_55",
        "energy_thrs": 1e-06,
        "file_chk": "checkpoint/phi_scf",
        "final_prec": 1e-06,
        "helmholtz_prec": -1.0,
        "kain": 10,
        "light_speed": -1.0,
        "localize": true,
        "max_iter": 200,
        "method": "DFT (PBE0)",
        "orbital_thrs": 4.9999999999999996e-05,
        "proj_prec": 1e-06,
        "relativity": "None",
        "rotation": 0,
        "shared_memory": true,
        "smooth_prec": 1e-06,
        "start_prec": 1e-06
      },
      "write_orbitals": {
        "file_phi_a": "orbitals/phi_a_scf",
        "file_phi_b": "orbitals/phi_b_scf",
        "file_phi_p": "orbitals/phi_p_scf"
      }
    },
    "schema_name": "mrchem_input",
    "schema_version": 1
  },
  "output": {
    "properties": {
      "center_of_mass": [
        0.0,
        0.0,
        3.4251476112129714e-16
      ],
      "charge": 0,
      "dipole_moment": {
        "dip-1": {
          "magnitude": 3.0376866058749776e-05,
          "r_O": [
            0.0,
            0.0,
            0.0
          ],
          "vector": [
            -4.801456028783325e-08,
            1.4388204755379423e-08,
            -3.0376824704580108e-05
          ],
          "vector_el": [
            -4.813723657717459e-08,
            1.426569690199023e-08,
            -3.0376824434573868e-05
          ],
          "vector_nuc": [
            1.2267628934133565e-10,
            1.2250785338919186e-10,
            0.0
          ]
        }
      },
      "geometry": [
        {
          "symbol": "Na",
          "xyz": [
            0.0,
            0.0,
            -2.9091386749999995
          ]
        },
        {
          "symbol": "Na",
          "xyz": [
            0.0,
            0.0,
            2.9091386750000003
          ]
        }
      ],
      "multiplicity": 1,
      "orbital_energies": {
        "energy": [
          -38.54642747424347,
          -38.54641775321415,
          -2.248443849763706,
          -2.248309653946132,
          -1.1612599241211912,
          -1.1603384437891975,
          -1.1603384428562953,
          -1.1600902276658376,
          -1.160090226729738,
          -1.1592715613192868,
          -0.13257133200533147,
          -38.54642770783908,
          -38.54641749479784,
          -2.2484438293969955,
          -2.248309646171431,
          -1.1612598864319814,
          -1.1603384341605887,
          -1.1603384334721436,
          -1.160090218563597,
          -1.1600902179766763,
          -1.1592715250821022,
          -0.13257113869732076
        ],
        "occupation": [
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0,
          1.0
        ],
        "spin": [
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "a",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b",
          "b"
        ],
        "sum_occupied": -177.3671174222441
      },
      "scf_energy": {
        "E_ee": 181.08544510682117,
        "E_eext": 0.0,
        "E_el": -345.1913335611999,
        "E_en": -821.3578138551799,
        "E_kin": 323.7311072133815,
        "E_next": 0.0,
        "E_nn": 20.7965335306678,
        "E_nuc": 20.7965335306678,
        "E_tot": -324.3948000305321,
        "E_x": -6.990453983303779,
        "E_xc": -21.659618042918897,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      }
    },
    "provenance": {
      "creator": "MRChem",
      "mpi_processes": 30,
      "nthreads": 6,
      "routine": "mrchem.x",
      "total_cores": 180,
      "version": "1.1.0-alpha"
    },
    "rsp_calculations": null,
    "scf_calculation": {
      "initial_energy": {
        "E_ee": 180.3969987297375,
        "E_eext": 0.0,
        "E_el": -343.13274881561887,
        "E_en": -814.9131319168391,
        "E_kin": 319.9031150353551,
        "E_next": 0.0,
        "E_nn": 20.7965335306678,
        "E_nuc": 20.7965335306678,
        "E_tot": -322.3362152849511,
        "E_x": -6.968369016822557,
        "E_xc": -21.551361647049767,
        "Er_el": 0.0,
        "Er_nuc": 0.0,
        "Er_tot": 0.0
      },
      "scf_solver": {
        "converged": true,
        "cycles": [
          {
            "energy_terms": {
              "E_ee": 180.62584369863288,
              "E_eext": 0.0,
              "E_el": -345.1128134142248,
              "E_en": -816.903766162178,
              "E_kin": 319.7237065758366,
              "E_next": 0.0,
              "E_nn": 20.7965335306678,
              "E_nuc": 20.7965335306678,
              "E_tot": -324.31627988355694,
              "E_x": -6.973981054615229,
              "E_xc": -21.584616471900958,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -324.31627988355694,
            "energy_update": 1.9800645986058498,
            "mo_residual": 0.23853507706860103,
            "wall_time": 90.389042603
          },
          {
            "energy_terms": {
              "E_ee": 180.6419201085619,
              "E_eext": 0.0,
              "E_el": -345.17883449845203,
              "E_en": -819.0265178175325,
              "E_kin": 321.78954039606805,
              "E_next": 0.0,
              "E_nn": 20.7965335306678,
              "E_nuc": 20.7965335306678,
              "E_tot": -324.3823009677842,
              "E_x": -6.976861559625478,
              "E_xc": -21.606915625923964,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -324.3823009677842,
            "energy_update": 0.06602108422725905,
            "mo_residual": 0.06056251191616136,
            "wall_time": 63.414121225
          },
          {
            "energy_terms": {
              "E_ee": 180.9388514399586,
              "E_eext": 0.0,
              "E_el": -345.1882829467879,
              "E_en": -820.4652088960211,
              "E_kin": 322.9670267619073,
              "E_next": 0.0,
              "E_nn": 20.7965335306678,
              "E_nuc": 20.7965335306678,
              "E_tot": -324.3917494161201,
              "E_x": -6.9867053511364325,
              "E_xc": -21.64224690149632,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -324.3917494161201,
            "energy_update": 0.009448448335888315,
            "mo_residual": 0.03038138773270655,
            "wall_time": 53.234372606
          },
          {
            "energy_terms": {
              "E_ee": 180.9773804875994,
              "E_eext": 0.0,
              "E_el": -345.19023075902726,
              "E_en": -820.7872432379215,
              "E_kin": 323.25501356211953,
              "E_next": 0.0,
              "E_nn": 20.7965335306678,
              "E_nuc": 20.7965335306678,
              "E_tot": -324.39369722835943,
              "E_x": -6.987768434249441,
              "E_xc": -21.64761313657517,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -324.39369722835943,
            "energy_update": 0.0019478122393365993,
            "mo_residual": 0.015476154141205062,
            "wall_time": 52.290403307
          },
          {
            "energy_terms": {
              "E_ee": 181.06408969495791,
              "E_eext": 0.0,
              "E_el": -345.1912914224211,
              "E_en": -821.3192473879366,
              "E_kin": 323.7122959606025,
              "E_next": 0.0,
              "E_nn": 20.7965335306678,
              "E_nuc": 20.7965335306678,
              "E_tot": -324.39475789175333,
              "E_x": -6.990202542131898,
              "E_xc": -21.658227147913003,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -324.39475789175333,
            "energy_update": 0.0010606633939005405,
            "mo_residual": 0.008656352198718138,
            "wall_time": 52.722205751
          },
          {
            "energy_terms": {
              "E_ee": 181.0831002827112,
              "E_eext": 0.0,
              "E_el": -345.191327593483,
              "E_en": -821.3766249588691,
              "E_kin": 323.7526829073966,
              "E_next": 0.0,
              "E_nn": 20.7965335306678,
              "E_nuc": 20.7965335306678,
              "E_tot": -324.3947940628152,
              "E_x": -6.990602275090754,
              "E_xc": -21.65988354963094,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -324.3947940628152,
            "energy_update": 3.6171061879031186e-05,
            "mo_residual": 0.0018637842876104026,
            "wall_time": 53.03473038
          },
          {
            "energy_terms": {
              "E_ee": 181.08660918511254,
              "E_eext": 0.0,
              "E_el": -345.1913330295756,
              "E_en": -821.3733471765944,
              "E_kin": 323.7459012368547,
              "E_next": 0.0,
              "E_nn": 20.7965335306678,
              "E_nuc": 20.7965335306678,
              "E_tot": -324.3947994989078,
              "E_x": -6.990566810042105,
              "E_xc": -21.65992946490637,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -324.3947994989078,
            "energy_update": 5.436092578747775e-06,
            "mo_residual": 0.0008850617408165189,
            "wall_time": 53.271208682
          },
          {
            "energy_terms": {
              "E_ee": 181.08638248536718,
              "E_eext": 0.0,
              "E_el": -345.19133345400047,
              "E_en": -821.3666740747327,
              "E_kin": 323.73923060657324,
              "E_next": 0.0,
              "E_nn": 20.7965335306678,
              "E_nuc": 20.7965335306678,
              "E_tot": -324.3947999233327,
              "E_x": -6.990497032489383,
              "E_xc": -21.65977543871884,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -324.3947999233327,
            "energy_update": 4.244249112161924e-07,
            "mo_residual": 0.0003474565301584773,
            "wall_time": 54.267750501
          },
          {
            "energy_terms": {
              "E_ee": 181.08578043237506,
              "E_eext": 0.0,
              "E_el": -345.1913335346651,
              "E_en": -821.360688270945,
              "E_kin": 323.7337043478884,
              "E_next": 0.0,
              "E_nn": 20.7965335306678,
              "E_nuc": 20.7965335306678,
              "E_tot": -324.39480000399726,
              "E_x": -6.990464391967239,
              "E_xc": -21.659665652016265,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -324.39480000399726,
            "energy_update": 8.066456302913139e-08,
            "mo_residual": 0.00013654053881460875,
            "wall_time": 56.460371629
          },
          {
            "energy_terms": {
              "E_ee": 181.08544510682117,
              "E_eext": 0.0,
              "E_el": -345.1913335611999,
              "E_en": -821.3578138551799,
              "E_kin": 323.7311072133815,
              "E_next": 0.0,
              "E_nn": 20.7965335306678,
              "E_nuc": 20.7965335306678,
              "E_tot": -324.3948000305321,
              "E_x": -6.990453983303779,
              "E_xc": -21.659618042918897,
              "Er_el": 0.0,
              "Er_nuc": 0.0,
              "Er_tot": 0.0
            },
            "energy_total": -324.3948000305321,
            "energy_update": 2.6534848984738346e-08,
            "mo_residual": 6.87729017964499e-05,
            "wall_time": 62.959624109
          }
        ],
        "wall_time": 592.109439003
      },
      "success": true
    },
    "schema_name": "mrchem_output",
    "schema_version": 1,
    "success": true
  }
}
