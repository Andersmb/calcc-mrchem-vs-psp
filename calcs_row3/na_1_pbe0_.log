Starting job 5688121 on c2-[4,45],c5-23,c10-[35,52] at Fri May 20 03:10:42 CEST 2022

Copying na_1_pbe0_.out to /cluster/home/ambr/projects/psp/row3/calcs/singlets
Copying na_1_pbe0_.json to /cluster/home/ambr/projects/psp/row3/calcs/singlets

Task and CPU usage stats:
JobID           JobName  AllocCPUS   NTasks     MinCPU MinCPUTask     AveCPU    Elapsed ExitCode 
------------ ---------- ---------- -------- ---------- ---------- ---------- ---------- -------- 
5688121      na_1_pbe0_        180                                             00:12:28      0:0 
5688121.bat+      batch         36        1   00:00:02          0   00:00:02   00:12:28      0:0 
5688121.ext+     extern        180        5   00:00:00          4   00:00:00   00:12:28      0:0 
5688121.0      mrchem.x        180       30   00:12:14         29   00:28:57   00:12:18      0:0 

Memory usage stats:
JobID            MaxRSS MaxRSSTask     AveRSS MaxPages   MaxPagesTask   AvePages 
------------ ---------- ---------- ---------- -------- -------------- ---------- 
5688121                                                                          
5688121.bat+     19280K          0     19280K        0              0          0 
5688121.ext+          0          4          0        0              4          0 
5688121.0      6208184K         29 3297242043        0             22          0 

Disk usage stats:
JobID         MaxDiskRead MaxDiskReadTask    AveDiskRead MaxDiskWrite MaxDiskWriteTask   AveDiskWrite 
------------ ------------ --------------- -------------- ------------ ---------------- -------------- 
5688121                                                                                               
5688121.bat+       16.90M               0         16.90M        0.03M                0          0.03M 
5688121.ext+        0.00M               4          0.00M            0                4              0 
5688121.0           7.32M              23          6.99M       67.01M               19         40.75M 

Job 5688121 completed at Fri May 20 03:23:11 CEST 2022
