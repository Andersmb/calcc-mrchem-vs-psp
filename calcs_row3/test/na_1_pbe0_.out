                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.1.0-alpha                              ***
***                                                                     ***
***         Git branch         HEAD                                     ***
***         Git commit hash    e7d4b230bb6df8495b9a                     ***
***         Git commit author  Stig Rune Jensen                         ***
***         Git commit date    Sat Mar 19 14:14:51 2022 +0100           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :        (1 bank)                              30
 OpenMP threads          :                                               6
 Total cores             :                                             175
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : MPI/OpenMP (6 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 9
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   32.00000    32.00000    32.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -32.00000   -32.00000   -32.00000 ]
 upper bounds          : [   32.00000    32.00000    32.00000 ]
 total length          : [   64.00000    64.00000    64.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Na            :        0.000000        0.000000       -2.909139
    1      Na            :        0.000000        0.000000        2.909139
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                 Computing Ground State Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-04
 Screening               : 1.20000e+01 StdDev
 Restricted              : False
 Functional              : LDA (SVWN5)
 AO basis                : 3-21G
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                              11
 Beta electrons          :                                              11
 Total electrons         :                                              22
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    1    a          :                              1.000000206722e+00
    1    1    a          :                              9.999999391869e-01
    2    1    a          :                              9.999939025377e-01
    3    1    a          :                              1.000005051268e+00
    4    1    a          :                              9.999876435549e-01
    5    1    a          :                              1.000004142982e+00
    6    1    a          :                              1.000004142982e+00
    7    1    a          :                              9.999948608116e-01
    8    1    a          :                              9.999943372158e-01
    9    1    a          :                              1.000006806954e+00
   10    1    a          :                              1.000000645904e+00
   11    1    b          :                              1.000001968269e+00
   12    1    b          :                              9.999987696087e-01
   13    1    b          :                              9.999934323763e-01
   14    1    b          :                              1.000005818432e+00
   15    1    b          :                              9.999936538910e-01
   16    1    b          :                              1.000004293556e+00
   17    1    b          :                              1.000004218213e+00
   18    1    b          :                              9.999954245349e-01
   19    1    b          :                              9.999954245349e-01
   20    1    b          :                              1.000005495503e+00
   21    1    b          :                              1.000000644273e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Precision               : 1.00000e-04
 Localization            : On
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                319.903115035355
 E-N energy              :            (au)               -814.913131916839
 Coulomb energy          :            (au)                180.396998729738
 Exchange energy         :            (au)                 -6.968369016823
 X-C energy              :            (au)                -21.551361647050
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 20.796533530668
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -343.132748815619
 Nuclear energy          :            (au)                 20.796533530668
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.223362152850e+02
                         :      (kcal/mol)             -2.022690272929e+05
                         :        (kJ/mol)             -8.462936107429e+05
                         :            (eV)             -8.771214869679e+03
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -38.320578919764
    1    1    a          :            (au)                -38.314581511984
    2    1    a          :            (au)                 -2.222164436643
    3    1    a          :            (au)                 -2.221055453337
    4    1    a          :            (au)                 -1.131316140672
    5    1    a          :            (au)                 -1.130528241010
    6    1    a          :            (au)                 -1.130527699220
    7    1    a          :            (au)                 -1.127798414461
    8    1    a          :            (au)                 -1.127798304479
    9    1    a          :            (au)                 -1.126697219886
   10    1    a          :            (au)                 -0.121455264737
   11    1    b          :            (au)                -38.320604751595
   12    1    b          :            (au)                -38.314558609930
   13    1    b          :            (au)                 -2.222192039129
   14    1    b          :            (au)                 -2.221035162893
   15    1    b          :            (au)                 -1.131394971911
   16    1    b          :            (au)                 -1.130537609475
   17    1    b          :            (au)                 -1.130524184719
   18    1    b          :            (au)                 -1.127800112913
   19    1    b          :            (au)                 -1.127785036179
   20    1    b          :            (au)                 -1.126617500200
   21    1    b          :            (au)                 -0.121457299551
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -175.949008884687
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Optimize ground state orbitals
 Method                  : DFT (PBE0)
 Relativity              : None
 Checkpointing           : Off
 Max iterations          : 200
 KAIN solver             : 10
 Localization            : First two iterations
 Diagonalization         : Off
 Start precision         : 1.00000e-06
 Final precision         : 1.00000e-06
 Helmholtz precision     : Dynamic
 Energy threshold        : 1.00000e-06
 Orbital threshold       : 5.00000e-05
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
 Iter           MO residual             Total energy                Update
---------------------------------------------------------------------------
    0          4.690416e+00        -322.336215284951         -3.223362e+02
    1          2.385351e-01        -324.316279883557         -1.980065e+00
    2          6.056251e-02        -324.382300967784         -6.602108e-02
    3          3.038139e-02        -324.391749416120         -9.448448e-03
    4          1.547615e-02        -324.393697228359         -1.947812e-03
    5          8.656352e-03        -324.394757891753         -1.060663e-03
    6          1.863784e-03        -324.394794062815         -3.617106e-05
    7          8.850617e-04        -324.394799498908         -5.436093e-06
    8          3.474565e-04        -324.394799923333         -4.244249e-07
    9          1.365405e-04        -324.394800003997         -8.066456e-08
   10          6.877290e-05        -324.394800030532         -2.653485e-08
---------------------------------------------------------------------------
                      SCF converged in 10 iterations!
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               0
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Na            :        0.000000        0.000000       -2.909139
    1      Na            :        0.000000        0.000000        2.909139
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                323.731107213382
 E-N energy              :            (au)               -821.357813855180
 Coulomb energy          :            (au)                181.085445106821
 Exchange energy         :            (au)                 -6.990453983304
 X-C energy              :            (au)                -21.659618042919
 Ext. field (el)         :            (au)                  0.000000000000
---------------------------------------------------------------------------
 N-N energy              :            (au)                 20.796533530668
 Ext. field (nuc)        :            (au)                  0.000000000000
 Tot. Reac. Energy       :            (au)                  0.000000000000
 El. Reac. Energy        :            (au)                  0.000000000000
 Nuc. Reac. Energy       :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)               -345.191333561200
 Nuclear energy          :            (au)                 20.796533530668
---------------------------------------------------------------------------
 Total energy            :            (au)             -3.243948000305e+02
                         :      (kcal/mol)             -2.035608087135e+05
                         :        (kJ/mol)             -8.516984242101e+05
                         :            (eV)             -8.827231811849e+03
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    1    a          :            (au)                -38.546427474243
    1    1    a          :            (au)                -38.546417753214
    2    1    a          :            (au)                 -2.248443849764
    3    1    a          :            (au)                 -2.248309653946
    4    1    a          :            (au)                 -1.161259924121
    5    1    a          :            (au)                 -1.160338443789
    6    1    a          :            (au)                 -1.160338442856
    7    1    a          :            (au)                 -1.160090227666
    8    1    a          :            (au)                 -1.160090226730
    9    1    a          :            (au)                 -1.159271561319
   10    1    a          :            (au)                 -0.132571332005
   11    1    b          :            (au)                -38.546427707839
   12    1    b          :            (au)                -38.546417494798
   13    1    b          :            (au)                 -2.248443829397
   14    1    b          :            (au)                 -2.248309646171
   15    1    b          :            (au)                 -1.161259886432
   16    1    b          :            (au)                 -1.160338434161
   17    1    b          :            (au)                 -1.160338433472
   18    1    b          :            (au)                 -1.160090218564
   19    1    b          :            (au)                 -1.160090217977
   20    1    b          :            (au)                 -1.159271525082
   21    1    b          :            (au)                 -0.132571138697
---------------------------------------------------------------------------
 Sum occupied            :            (au)               -177.367117422244
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000        0.000000       -0.000030
 Magnitude               :            (au)                        0.000030
                         :         (Debye)                        0.000077
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :       -0.000000        0.000000       -0.000030
 Magnitude               :            (au)                        0.000030
                         :         (Debye)                        0.000077
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h 12m 14s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
